artifact/rules.bzl

```diff
@@ -51,11 +51,11 @@ def _deploy_artifact_impl(ctx):
     symlinks = {
         'VERSION': version_file,
     }
-
+    cloudsmith_lib_files = ctx.attr._cloudsmith_pylib[DefaultInfo].default_runfiles.files.to_list()
     return DefaultInfo(
         executable = _deploy_script,
         runfiles = ctx.runfiles(
-            files = files,
+            files = files + cloudsmith_lib_files,
```

Comment:
How we add the dependency on the `//common/cloudsmith:clodsmith-wrapper` wrapper to the script.